### DISCUSSION TITLE
Pretrained KIE Models for doctr

### DISCUSSION THREAD

**Discussion by ffalkenberg** [mindee/doctr#1369] [General]
**Description**:

Hello everyone! 🚀

I've been have been particularly interested in Key Information Extraction (KIE). Before diving into training a model from scratch, I wanted to check if there are any pretrained KIE models suitable for `doctr`. Pretrained models could save a lot of time and effort. I would be especially interested in: date, person and price information. 

**Questions**:
1. Is there some place where i can find pretrained KIE models that would be easy to use in  `doctr`
1. Has anyone here experimented with pretrained KIE models for `doctr`?
2. If yes, could you share your experience?

Any pointers or shared experiences would be highly appreciated!

Thanks in advance! 🙏

**Comment by felixT2K** [comment]
Hi @ffalkenberg 👋🏼,


Unfortunately we don't provide pretrained KIE models atm in doctr :/

Maybe @aminemindee can provide some help / insights ?

**Reply by jbesq-ml** [reply]
updated:: 
https://github.com/mindee/doctr/discussions/1682
